habitat_baselines/config/default.py

```diff
@@ -65,7 +65,7 @@
 # The name of the project on W&B.
 _C.WB.PROJECT_NAME = ""
 # Logging entity (like your username or team name)
-_C.WB.ENTITY = ""
+_C.WB.ENTITY = "language-rearrangement"
```

Comment:
You will also need to make the following change in this file for wandb to work properly:
```_C.WRITER_TYPE = "wb"```
```_C.WB.PROJECT_NAME = "object_rearrangement"```

Reply:
Also, it is recommended that you use a novel `WB.RUN_NAME` for every run that you conduct so its easier to visualize the results

Reply:
Not sure it should be as default value for whole the Habitat Baselines package.

Reply:
Hey Oleksandr, we dont plan to merge this PR into main. This PR is to just track the changes we are doing on our side